src/hotspot/share/classfile/systemDictionaryShared.cpp

```diff
@@ -2278,9 +2291,10 @@ class SharedLambdaDictionaryPrinter : StackObj {
       ResourceMark rm;
       Klass* k = record->proxy_klass_head();
       while (k != nullptr) {
-        _st->print_cr("%4d: %s %s", (++_index), k->external_name(),
+        _st->print_cr("%4d: %s %s", _index, k->external_name(),
                       class_loader_name_for_shared(k));
         k = k->next_link();
+        _index++;
```

Comment:
Not clear why the _index handling is changing.

Reply:
I reverted this part. The problem is when construct the lambda dictionary library, I wrongly add one to the last dictionary's index.